src/generator.js

```diff
@@ -0,0 +1,59 @@
+const {Student, Criteria, LearningStyles, Subject, SubjectPreference} = require("./group");
+const fs = require("fs");
+
+let argv = process.argv.splice(2);
+
+/**
+ * @summary A function used to generate a randomized student, these will be destributed into groups
+ * @returns {Array} returns an array of students.
+ */
+function studentGenerator(){
+    let learningStyle;
+    let criteria;
+    let studentArray = [];
+    for (let index = 0; index < argv[0]; index++) {
+        learningStyle = new LearningStyles(lsGenerator(), lsGenerator(), lsGenerator(), lsGenerator());
```

Comment:
deklarer variablerne learningStyle og criteria her i stedet for at deklarere dem øverst og sætte dem senere

Reply:
Og de skal være const